### PR TITLE
[Merged by Bors] - Fix GHA Automated Builds

### PR DESCRIPTION
**Pull request by web3rover**
Currently the release builds are failing because protoc requires $GOPATH/bin to be added to $PATH. This PR adds this path so that protoc binaries can be found during `make build`

**Note that for builds to work the version.txt file shouldn't have a new line character.**

### CODE REVIEW

File: .github/workflows/release.yml
Comment (noamnelke) [inline .github/workflows/release.yml]: why remove the end-of-file line break?

**Review by noamnelke — APPROVED** [review]
Let's restore the end-of-file line break before merging?

**Review by noamnelke — CHANGES_REQUESTED** [review]
Actually... just noticed the comment... if you don't want a line break in `version.txt` - then let's fix it here.

**Review by noamnelke — APPROVED** [review]

**Comment by web3rover** [comment]
bors merge

**Comment by lrettig** [comment]
protoc is no longer used in production - it was removed a long time ago - it's only needed for the current build cycle because it's using very old code. Should this change be reverted?

**Comment by noamnelke** [comment]
once the testnet is caught up with the new code we can consider it, but I think puting `$GOPATH/bin` in `$PATH` is standard practice, so it's not a big deal if it stays this way, isn't it?

**Comment by lrettig** [comment]
> once the testnet is caught up with the new code we can consider it, but I think puting `$GOPATH/bin` in `$PATH` is standard practice, so it's not a big deal if it stays this way, isn't it?

True, it's probably no big deal, but the `ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'` caught my eye and I'm not 100% sure it's something we want in production forever - I guess that's up to @narayanprusty. This change _could_ be removed from `develop` and just cherry-picked into future testnet builds (until testnet catches up with the new API code).

**Comment by noamnelke** [comment]
>This change _could_ be removed from `develop` and just cherry-picked into future testnet builds (until testnet catches up with the new API code).

Yes, I agree.

@narayanprusty WDYT?

We could create a PR that reverts this now and merge it, since it's already in the current testnet and future testnets will either be based on the current one, or on `develop` once the SMApp supports API v2 (and then we don't need this change anyway).

**Comment by web3rover** [comment]
Having $GOPATH/bin in $PATH is already deprecated. https://blog.golang.org/modules2019 As we have protoc in old code therefore we needed it. 

`ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'` doesn't cause any security issues as it's defined for a specific step and not at global level. 

Regardless of that, I can create another PR to revert it and re-open this PR so that we can cherry-pick this PR until we have old code.

**Comment by web3rover** [comment]
@noamnelke @lrettig Here is the PR to revert the code: #2265

**Comment by noamnelke** [comment]
No need to reopen this PR - approved the other one.

**Pull request closed without merging**
